templates/registration/profile.html

```diff
@@ -1,6 +1,11 @@
 {% extends "base.html" %}
+{% load staticfiles %}
 {% load humanize %}
 
+{% block extrastyle %}
+    <link rel="stylesheet" href="{% static "css/artist/artist_list.css" %}" type="text/css" charset="utf-8" />
```

Comment:
Instead of reusing the styles from artist_list.css, it may make sense to find the common styles between the artist list page and this page and put them into a common file, which can then be applied to both artist_list.html and profile.html, or they could go in perdiem.css if you think they might be used elsewhere.